assets/js/components/user-input/UserInputPreview.js

```diff
@@ -48,15 +50,44 @@ export default function UserInputPreview( props ) {
 		error,
 	} = props;
 
+	const [ single ] = useQueryArg( 'single', false );
+	const [ question ] = useQueryArg( 'question', false );
 	const settings = useSelect( ( select ) => select( CORE_USER ).getUserInputSettings() );
 
+	useEffect( () => {
+		trackEvent( 'user_input', 'summary_view' );
+	}, [] );
+
 	const {
 		USER_INPUT_ANSWERS_GOALS,
 		USER_INPUT_ANSWERS_HELP_NEEDED,
 		USER_INPUT_ANSWERS_POST_FREQUENCY,
 		USER_INPUT_ANSWERS_ROLE,
 	} = getUserInputAnwsers();
 
+	const isSettings = single === 'settings';
+
+	const handleGoTo = ( num, singleType, slug ) => {
+		trackEvent(
+			'user_input',
+			isSettings ? 'settings_edit' : 'summary_edit',
+			slug
+		);
+		goTo( num, singleType );
+	};
+
+	const handleSubmit = () => {
+		trackEvent(
+			'user_input',
+			isSettings ? 'summary_submit' : 'question_submit',
+			question,
+		);
```

Comment:
@felixarntz I've reverted the changes to `UserInputPreview` and tracked all the events in the parent component `UserInputQuestionnaire` I suggest we keep the `noFooter` prop since we explicitly want to no keep the footer. If we relied on the `isSettings` variable, it just checked whether the `single` query var was set to `settings` and this wouldn't work in our stories.